pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -44,44 +38,59 @@ buildDotnetModule rec {
 
   preConfigure = ''
     substituteInPlace Directory.Build.props \
-      --replace '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
+      --replace-fail '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
   '';
 
-  postPatch = ''
-    ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
-  '';
+  selfContainedBuild = true;
+
+  dotnetBuildFlags = [
+    # Tell the app it is a distro package; affects wording in update prompts
+    "--property:INSTALLATION_METHOD_PACKAGE_MANAGER=1"
```

Comment:
This looks like an env var, shouldn't it be set in the wrapper instead of here?

Reply:
It is a build prop

Reply:
Oh my bad, didn't check previous conversations. Really confusing from the Nexus App folks.